.github/workflows/test.yml

```diff
@@ -84,3 +82,54 @@ jobs:
       - name: CTest
         run: ctest --no-tests=error -VV --build-config Release
         working-directory: ./out
+
+  alpine-alt-arch-makefile-test:
+    name: makefile-alpine-${{ matrix.arch }}-${{ matrix.cc }}
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        arch: [armv7, aarch64, s390x, ppc64le]
+        cc: [gcc, clang]
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - uses: uraimo/run-on-arch-action@v2
+        with:
+          arch: ${{matrix.arch}}
+          distro: alpine_latest
```

Comment:
Why does this need to run on Alpine Linux instead of on `ubuntu-latest`? Seems like extra overhead to me.

Reply:
@aklomp, no overhead, quite the opposite in fact since  `uraimo/run-on-arch-action@v2` pulls images from docker hub, alpine images are 5.5 MB, ubuntu images are 65 MB. The size of installed packages shall be roughly the same.

Reply:
Nice, that sounds worthwhile. I would prefer if this sort of rationale is mentioned in the commit body, though. (I can add some text to the commit body when merging, if you like.)

Reply:
I will amend the commit message.